server/routes/rankingRedudanceGroupLength.ts

```diff
@@ -0,0 +1,19 @@
+import { Request, Response } from "express";
+import { sqlUtils } from "@/db";
+
+export default async function getRankingRedudanceGroupLength(
```

Comment:
Umbenennung in `getRedundanceGroupLengthRanking`?

Reply:
Ich hab das Ranking komplett weggelassen. weil das Ranking sich ja nur auf die redundanz bezieht und nicht auf die Grouplength. Bei den anderen Routen zu keyvalue steht auch kein ranking dahinter, deshalb habe ich es ganz weggelassen.